controllers/kserve_inferenceservice_controller_test.go

```diff
@@ -95,6 +103,89 @@ var _ = Describe("The Openshift Kserve model controller", func() {
 			}, timeout, interval).Should(Succeed())
 		})
 
+		It("With a new Kserve InferenceService, serving cert annotation should be added to the runtime Service object.", func() {
+			// Create a new InferenceService
+			inferenceService := &kservev1beta1.InferenceService{}
+			err := convertToStructuredResource(KserveInferenceServicePath1, inferenceService)
+			Expect(err).NotTo(HaveOccurred())
+			inferenceService.SetNamespace(testNs)
+			Expect(cli.Create(ctx, inferenceService)).Should(Succeed())
+
+			// Stub: Create a Runtime Service, which must be created by the OpenDataHub operator.
```

Comment:
Comment seems confusing... This is not a Runtime Service, but a KServe service.

Reply:
I intended to represent a Service created for each ISVC as "Runtime Service." It seems clearer to rename it to "Kserve Service," so I will proceed with that change.